sample-plugin/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Weird but OK.

Reply:
This is the powershell xml module doing this. The XML spec says it is case insentitive. Yet when the spec talks about UTF-8 it is usually uppercased but this works according to spec.